cmd/istioctl/main.go

```diff
@@ -301,7 +301,7 @@ func init() {
 	}
 	rootCmd.PersistentFlags().StringVarP(&kubeconfig, "kubeconfig", "c", defaultKubeconfig,
 		"Kubernetes configuration file")
-	rootCmd.PersistentFlags().StringVarP(&istioSystem, "namespace", "n", v1.NamespaceDefault,
+	rootCmd.PersistentFlags().StringVarP(&istioSystem, "namespace", "n", kube.IstioNamespace,
```

Comment:
I think the meaning of this flag changed with CRDs to be the place where CRDs are stored. this has effect on rule match conditions.